care/users/migrations/0022_user_mfa_settings_user_totp_secret.py

```
@@ -0,0 +1,23 @@
+        ('users', '0021_rename_gender_user_old_gender_user_geo_organization_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='mfa_settings',
+            field=models.JSONField(blank=True, default=dict),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='totp_secret',
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)